fractal_server/app/routes/api/v2/_aux_functions_tasks.py

```diff
@@ -226,11 +255,18 @@ async def _verify_non_duplication_user_constraint(
     res = await db.execute(stm)
     duplicate = res.scalars().all()
     if duplicate:
+        if len(duplicate) > 1:
+            logger.error(
+                "Found more then one TaskGroupV2 with "
+                f"({user_id=}, {pkg_name=}, {version=})."
+            )
```

Comment:
On top of logging, we raise a 422 with message
 f"User '{user.email}' already owns {N} task groups with name={pkg_name} and {version=}.\n"
"Warning: this should have not happened, please contact an admin."